templates/recording-deployment.yaml

```diff
@@ -87,3 +90,7 @@ spec:
           runAsUser: {{ (.Values.recording.runAs).user | default "1000" }}
       imagePullSecrets:
       - name: {{ .Release.Name }}-docker-cfg
+      volumes:
+      - name: cache
+        persistentVolumeClaim:
+          claimName: {{ .Release.Name }}-sockets-pvc
```

Comment:
Re-using the sockets drive here, which you briefly mentioned. Not sure if we want to create a new claim for this though but seems like a lot of overhead? Since this is only a 24hr cache maybe it's fine to do this.